### DISCUSSION TITLE
RustDesk Android Telemetry/Spyware?

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#5753] [Q&A]
On the RustDesk F-Droid page (https://f-droid.org/en/packages/com.carriez.flutter_hbb/) it says:
"This app tracks and reports your activity"
What does this mean? Is there telemetry or any other kind of spyware in the RustDesk Android app?

**Comment by SergeyMy** [comment]
Flutter from Google, it has telemetry out of the box, but the assembly does not work for me on Linux without the command:
**--disable-telemetry     Disable telemetry reporting when this command runs.**
PS
It seems to me that assembly with telemetry requires additional components, I don’t see them in the assembly scripts.
